### PR TITLE
create 'asf-site' and redirect files

### PR DESCRIPTION
**Pull request by sreev**
In, https://github.com/apache/incubator-heron
add new branch name 'asf-site'
create two files
.htaccess
index.html
(both these files redirects to https://apache.github.io/incubator-heron/)

https://issues.apache.org/jira/browse/INFRA-16118
https://github.com/apache/incubator-heron/issues/2758

### CODE REVIEW

**Comment by sreev** [comment]
please pay attention to this file when merging and discard this version of this file.
heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java

**Comment by sreev** [comment]
'asf-site' branch should exists in apache.org/incubator-heron.

**Comment by kramasamy** [comment]
@sreev - the changes are not just for asf-site - other code files have been changed as well. Can you remove those changes?

File: heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java [hunk 1]
```diff
@@ -1,264 +0,0 @@
-// Copyright 2018 Twitter. All rights reserved.
```
Comment (nwangtw) [inline heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java]: why this file is here?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+    <META HTTP-EQUIV=Refresh CONTENT="5; URL=http://www.example.org">
```
Comment (nwangtw) [inline index.html]: why refresh?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  </head>
+  <body>
+    The url of this site has been changed. Please update your bookmarks!
```
Comment (nwangtw) [inline index.html]: maybe "changed to http://......."? I am wondering if apache has a standard page with extra features like a better design, localization, etc.

**Comment by kramasamy** [comment]
@sreev - should we remove this file? @huijunwu might be needing this.

File: heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java (hunk 1 above)
Comment (sreev) [reply heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java]: it came in from merge conflict.

CONFLICT (rename/delete): heron/healthmgr/src/java/com/twitter/heron/healthmgr/HealthManagerMetrics.java deleted in f444372de2749c8a6953ff98a5178550ec026cd8 and renamed to heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java in HEAD. Version HEAD of heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java left in tree.


Deleted merge conflict for 'heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java':
  {local}: created file
  {remote}: deleted
Use (c)reated or (d)eleted file, or (a)bort? d

File: index.html (hunk 2 above)
Comment (nwangtw) [reply index.html]: kk thx.

**Comment by sreev** [comment]
i will create another pull request with more cleaner commit.

**Pull request closed without merging**